scripts/debug_tools/prepare_all_cmd_for_ctu.py

```diff
@@ -79,13 +80,13 @@ def get_triple_arch(analyze_command_file):
         help="Path to the used clang plugin.")
     args = parser.parse_args()
 
-    compile_cmd_debug = "compile_cmd_DEBUG.json"
+    compile_cmd_debug = "compilation_database_DEBUG.json"
     with open(compile_cmd_debug, 'w',
               encoding="utf-8", errors="ignore") as f:
         f.write(
             json.dumps(
                 prepare_compile_cmd.prepare(
-                    os.path.join(args.report_dir, "compile_cmd.json"),
+                    os.path.join(args.report_dir, "compilation_database.json"),
```

Comment:
Here you expect `compilation_database.json` in the report directory but CodeChecker will create a `compile_cmd.json` to the report directory when you run the `CodeChecker analyze` command.

See:
https://github.com/Ericsson/codechecker/blob/65f8a5056ee6e86661ae799708f8ec4392a5e67c/analyzer/codechecker_analyzer/cmd/analyze.py#L1037

Reply:
Well, then we should consider updating that behavior to prefer using `compilation_database.json`

Reply:
I decided to support the `compile_cmd.json` via moving it to `compilation_database.json`.
It's basically canonicalizing to the latter.